tests/test_api.py

```diff
@@ -576,9 +575,61 @@ def test_get_datatype_manifest(self, client, syn_token):
                 "Year of Birth": "Int64",
                 "entityId": "string"}
 
+    # small manifest: syn51078535; big manifest: syn51156998
+    @pytest.mark.parametrize("manifest_id",["syn51156998", "syn51156998"]) 
+    @pytest.mark.parametrize("new_manifest_name",["Example", None]) 
+    @pytest.mark.parametrize("as_json",[True, False, None]) 
+    def test_manifest_download(self, config, client, syn_token, manifest_id, new_manifest_name, as_json):
+        params = {
+            "input_token": syn_token,
+            "manifest_id": manifest_id,
+            "new_manifest_name": new_manifest_name, 
+            "as_json": as_json
+
+        }
+
+        response = client.get('http://localhost:3001/v1/manifest/download', query_string = params)
```

Comment:
Nit: it might make your life easier if you had a global config for (maybe a Jira ticket to refactor)

```
SERVER_URL = "http://localhost:3001/v1"
```

Then you can do

```
f"{SERVER_URL}/manifest/download"
```

Reply:
Thanks Tom. I will add the TO DO like you suggested.